src/Illuminate/Console/Command.php

```diff
@@ -189,7 +197,88 @@ public function run(InputInterface $input, OutputInterface $output)
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        return $this->laravel->call([$this, 'handle']);
+        if ($this->hasValidInput()) {
+            return $this->laravel->call([$this, 'handle']);
+        }
+
+        return $this->displayValidationErrors();
+    }
+
+    /**
+     * Check if the argument and option values are valid.
+     *
+     * @return bool
+     */
+    protected function hasValidInput()
+    {
+        if (empty($rules = $this->rules())) {
+            return true;
+        }
+
+        $this->validator = $this->validator ?? $this->makeValidator()->make(
```

Comment:
If the `makeValidator` method throws `\Illuminate\Contracts\Container\BindingResolutionException` (when the `illuminate/console` package is used outside of a full Laravel framework app) we should just return `true` in the `hasValidInput` method.

Reply:
Done.